### DISCUSSION TITLE
Filter for invoice listing

### DISCUSSION THREAD

**Discussion by lamp4youde** [kimai/kimai#2425] [General]
Hi,

it would be nice to have a filter in the invoice listing page:

![image](https://user-images.githubusercontent.com/77755099/111024864-31b8b180-83e1-11eb-9519-13aba62d71ee.png)

So I could see only due invoices or customer X...

Is there a plan for this feature?

Thank you

Heiko

**Comment by kevinpapst** [comment]
Check the development demo

**Reply by lamp4youde** [reply]
Thank you - perfect - this is what I am searching for.

Any schedule for this as release?
